static/app/components/confirm.spec.jsx

```diff
@@ -101,12 +101,11 @@ describe('Confirm', function () {
 
     expect(mock).not.toHaveBeenCalled();
 
-    await userEvent.click(screen.getByRole('button'));
+    await userEvent.click(screen.getByRole('button', {name: 'Confirm?'}));
 
     // Click "Confirm" button, should be last button
     await userEvent.click(screen.getByText('Confirm'));
 
-    expect(screen.getByRole('dialog')).toBeInTheDocument();
```

Comment:
this is a good example of what disabled animations prevents. the dialog was still on the screen most of the time because of the animation.